app/src/main/res/layout/content_event.xml

```diff
@@ -23,7 +23,7 @@
             app:layout_constraintHorizontal_bias="0.5"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            tools:srcCompat="@drawable/placeholder" />
```

Comment:
there is no need of removing this line. This to display the placeholder in the design screen